adafruit_pn532/adafruit_pn532.py

```diff
@@ -380,7 +403,7 @@ def listen_for_passive_target(self, card_baud=_MIFARE_ISO14443A, timeout=1):
             return False  # _COMMAND_INLISTPASSIVETARGET failed
         return response
 
-    def get_passive_target(self, timeout=1):
+    def get_passive_target(self, timeout: float = 1) -> bytearray:
```

Comment:
This can return `None` and can otherwise return `Union[bytes, bytearray]`, so it's return should be wrapped with `Optional[Union[bytes, bytearray]]`.